src/utils.js

```diff
@@ -4,6 +4,7 @@
 import { __ } from "@wordpress/i18n";
 import { useSelect } from "@wordpress/data";
 import { store as coreStore } from "@wordpress/core-data";
+import { addQueryArgs } from "@wordpress/url";
```

Comment:
@kkoppenhaver It doesn't look this is actually being used. Shall we remove this?